data/en/mbs_0817.json

```diff
@@ -0,0 +1,390 @@
+                                            "commission",
+                                            "sales of goods purchased for resale",
+                                            "revenue earned from other parts of the business not named (please supply at fair value)"
+                                        ]
+                                    },
+                                    {
+                                        "title": "Exclude:",
+                                        "description": "",
+                                        "list": [
+                                            "VAT",
+                                            "income from the sale of fixed capital assets",
+                                            "grants and subsidies",
+                                            "insurance claims",
+                                            "interest received"
+                                        ]
+                                    }
+                                ]
+                            },
+                            "id": "turnover-question",
+                            "description": "",
+                            "answers": [{
+                                "id": "turnover-answer",
+                                "label": "Total turnover excluding VAT",
+                                "description": "",
+                                "type": "Currency",
+                                "currency": "GBP",
+                                "decimal_places": 2,
+                                "q_code": "40",
+                                "mandatory": true
+                            }],
+                            "type": "General",
+                            "title": "For the period {{ format_conditional_date (answers['reporting-period-answer-from'], exercise.start_date)}} to {{ format_conditional_date (answers['reporting-period-answer-to'], exercise.end_date)}}, what was the value of {{respondent.trad_as_or_ru_name}}’s <em>total turnover</em>, excluding VAT?"
+                        }],
+                        "title": "Turnover"
+                    },
+                    {
+                        "id": "changes-in-turnover-block",
+                        "type": "Question",
+                        "questions": [{
+                            "title": "Did any significant changes occur to the turnover for {{respondent.trad_as_or_ru_name}}?",
+                            "id": "changes-in-turnover-question",
+                            "description": "<p>Please note: what constitutes a ‘significant change’ is dependent on your own interpretation in relation to {{respondent.trad_as_or_ru_name}}’s figures from the previous reporting period and the same reporting period last year. </p><p>This information will help us to validate your data and should reduce the need to query any figures with you. </p>",
```

Comment:
In surveys we have a mix of `‘`, `’`, `'` and even `&apos;` (lfs, my bad - I think author made me do it), as well as `”`, `"` and `\"`, not sure it matters but just thought I'd point out.

Basically - I suspect the unicode `‘`, `’`, `”` etc are coming from a cut and paste from another document and I think we should clear on what is _meant_ to be in the schema.

Reply:
Correct apostrophe should always be `‘`. Ben showed me a description as to why but I can't remember where it was.

Here's a previous fix for reference https://github.com/ONSdigital/eq-survey-runner/issues/1458

Reply:
Should always be smart quotes I believe